android/src/main/kotlin/com/situm/situm_flutter/Extensions.kt

```diff
@@ -72,6 +90,27 @@ fun LocationRequest.Builder.fromArguments(args: Map<String, Any>): LocationReque
         }
         outdoorLocationOptions(outdoorLocationOptions.build())
     }
+    if (args.containsKey("useWifi")) {
+        val useWifi = args["useWifi"] as Boolean?
+        if (useWifi != null) {
```

Comment:
Una chorrada pero puede quedar mas simple con `args["useWifi"]?.let{ useWifi(it)}`

Reply:
Feito!